client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -29,7 +29,7 @@ const ciraAgreementUrl = 'https://services.cira.ca/agree/agreement/agreementVers
 const defaultValues = {
 	lang: 'EN',
 	legalType: 'CCT',
-	ciraAgreementAccepted: false,
+	ciraAgreementAccepted: true,
```

Comment:
I ended up reverting (well, hard resetting this) out as it felt weird to me to pre-check. I ended up running it past a8c legal in the end too as I second guessed myself after a bit and their preference was unchecked.